guides/doc-Upgrading_Project/topics/proc_upgrading-a-disconnected-project-server.adoc

```diff
@@ -62,6 +62,28 @@ endif::[]
 # {foreman-installer} --foreman-proxy-dns-managed=false \
 --foreman-proxy-dhcp-managed=false
 ----
+ifdef::katello,satellite,orcharino[]
```

Comment:
@AkshayGadhaveRH Also here. We explicitly mention the applicable builds here but this whole file is `Satellite` specific.

Reply:
@evgeni What can be done here? Disconnected Guide is Satellite-exclusive as of now. We can remove the above condition for this PR at least.

Reply:
I can drop it completely, sure.

Reply:
But, fwiw, we already have conditionals in this file:
https://github.com/theforeman/foreman-documentation/blob/6021389e1d5d4dadfe7da9279a2bb63f37ffc37f/guides/doc-Upgrading_Project/topics/proc_upgrading-a-disconnected-project-server.adoc?plain=1#L14

Reply:
Yes, this will be fixed soon. Thank you :+1: